### DISCUSSION TITLE
Guidance on service:telemetry:metrics setup

### DISCUSSION THREAD

**Discussion by julia320** [open-telemetry/opentelemetry-collector-contrib#20729] [General]
### Describe the issue you're reporting

Hi team,

I have a question regarding the setup for service telemetry metrics (discussed [here](https://opentelemetry.io/docs/collector/configuration/#service) and implemented [here](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L99-L111). I have added this to my pipeline and am able to view the metrics at https://localhost:8888/metrics. I have two questions regarding this - 

1. The comments in the implementation indicate that this feature is experimental and subject to removal. However, it seems to have been in place for some time, so I was wondering if this is still the case or if it can be relied on for production-level code. If there are plans to deprecate it, is there any alternative in place that can be used?

2. As mentioned, I am able to view the metrics at the local port 8888/metrics. But I'd like to export these metrics to some external location (final location is AWS cloudwatch, but if I can send them to some log file or anything similar I can upload this to cloudwatch separately). Is there any way to send the metrics to a location other than a local port? The only option as shown in the struct is an address, but is it possible to send this through another pipeline or exporter? My service is containerized so the local port is not reliably accessible to begin with. 

Any help you can provide is greatly appreciated :)

**Comment by atoulme** [comment]
1. There is no alternative that I'm aware of.
2. You can scrape this endpoint with the collector using the prometheus receiver and send the data points to cloudwatch using a pipeline.

**Comment by julia320** [comment]
Thanks for the response! Can you confirm whether there are plans to deprecate this feature per the comment in the implementation?
